features/db.feature

```diff
@@ -61,6 +61,18 @@ Feature: Perform database operations
       Success: Exported
       """
 
+    When I run `wp db export /tmp/wp-cli-behat.sql --porcelain`
```

Comment:
To prevent this test from being dependent on `/tmp/`, let's drop it from the filepath.